src/components/UI/Card/Card.jsx

```diff
@@ -70,8 +78,14 @@ const Card = ({ id, cardText, isViewMode }) => {
     });
   };
 
+  const openCardPage = () => {
+    if (!isEdited) {
+      navigate(`/card/${id}`);
+    }
+  };
```

Comment:
At this point event will still be triggered. Better do not assign it at all if card is in view mode.